src/Apis.js

```diff
@@ -1,5 +1,11 @@
-export default function getAllReviews() {
+export function getAllReviews() {
   return fetch("https://ezisberta-be-nc-games.herokuapp.com/api/reviews")
     .then((res) => res.json())
     .then((data) => data.reviews);
 }
+
+export function getCategories() {
+  return fetch("https://ezisberta-be-nc-games.herokuapp.com/api/categories")
```

Comment:
Thanks for the feedback! I will definitely consider a nav bar, probably for the desktop version.
I'll also look into the axios docs because it has completely vanished from my memory -.- , it shouldn't be a difficult replacement once I get how the functions should be written :)